.github/workflows/web-service.yml

```diff
@@ -2,12 +2,20 @@ name: Web Service Release
 
 on:
   workflow_dispatch:
+    inputs:
+      data_json_url:
+        required: true
+        description: A json file url with the data needed for the service (i.e. pastebin.pl)
+      tag:
+        required: true
```

Comment:
wait, this is still required, please change that to `false`

Reply:
❓ after the latest push this is in there again, does this need to be resolved, @biernacki ?